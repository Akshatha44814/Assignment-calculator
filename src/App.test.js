import { render } from "@testing-library/react";
import App from "./App";
import Calculator from "./Calculator/Calculator";

test("renders App Component", () => {
  render(<App />);
});

describe("renders Calculate Component", () => {
  it("render Calculate component", () => {
    render(<Calculator />);
  });
});