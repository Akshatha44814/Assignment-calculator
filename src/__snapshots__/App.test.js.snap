// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Calculate Component display the format label 1`] = `
<DocumentFragment>
  <p>
    (Format:
    <span>
      //[delimiter]
[numbers]
    </span>
    )
  </p>
</DocumentFragment>
`;

exports[`renders Calculate Component render Enter numbers label 1`] = `
<DocumentFragment>
  <div
    class="conatiner"
  >
    <h1>
      String Calculator
    </h1>
    <p
      class="content"
    >
      Enter numbers
    </p>
    <p
      class="content"
    >
      (Format:
      <span
        class="colorContent"
        data-testid="redColor"
      >
        //[delimiter]
[numbers]
      </span>
      )
    </p>
    <div
      class="conatinerPage"
    >
      <div>
        <input
          class="input"
          placeholder="E.G.,//;\\\\n1;2;3 or 1,2,3"
          type="text"
        />
      </div>
      <button
        class="calculate"
      >
        Calculate
      </button>
      <div
        class="resultInput"
      >
        <input
          class="input"
          disabled=""
          placeholder="Result Output"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`renders Calculate Component render String Calculator header 1`] = `
<DocumentFragment>
  <div
    class="conatiner"
  >
    <h1>
      String Calculator
    </h1>
    <p
      class="content"
    >
      Enter numbers
    </p>
    <p
      class="content"
    >
      (Format:
      <span
        class="colorContent"
        data-testid="redColor"
      >
        //[delimiter]
[numbers]
      </span>
      )
    </p>
    <div
      class="conatinerPage"
    >
      <div>
        <input
          class="input"
          placeholder="E.G.,//;\\\\n1;2;3 or 1,2,3"
          type="text"
        />
      </div>
      <button
        class="calculate"
      >
        Calculate
      </button>
      <div
        class="resultInput"
      >
        <input
          class="input"
          disabled=""
          placeholder="Result Output"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
